File: Modul-6/Soal-1/PRAK601-2410817220012-NOOR-KHALISA.py
baris, kolom = map(int, input("Baris dan kolom: ").split())
elemen = list(map(int, input("Elemen-elemen matriks: ").split()))

matriks = [elemen[i * kolom: (i + 1) * kolom] for i in range(baris)]
print("Matriks: ")
for row in matriks:
    print(" ".join(map(str, row)))

File: Modul-6/Soal-1/PRAK601-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(void){
    int baris, kolom;
    printf("Baris dan kolom: ");
    scanf("%d %d", &baris, &kolom);
    
    int matriks[baris][kolom];
    printf("Elemen-elemen matriks: ");
    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            scanf("%d", &matriks[i][j]);
        }
    }
    printf("Matriks: \n");
    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}